pkg/build/node/node.go

```diff
@@ -373,6 +379,12 @@ func (c *BuildContext) prePullImages(dir, containerID string) error {
 			movePulled = append(movePulled, fmt.Sprintf("/build/bits/images/%s", pullName))
 		}
 	}
+
+	// Create the /kind/images directory inside the container.
+	if err = execInBuild("mkdir", "-p", "/kind/images/"); err != nil {
```

Comment:
would you mind adding a TODO about pulling "/kind/images/" out to be in a constant?